sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -2383,6 +2383,13 @@ val SHUFFLE_COMPRESSION_LZ4_CHUNK_SIZE = conf("spark.rapids.shuffle.compression.
       .booleanConf
       .createWithDefault(true)
 
+  val TRACE_TASK_GPU_OWNERSHIP = conf("spark.rapids.sql.traceTaskGpuOwnership")
```

Comment:
suggestion: be very specific about this config. I'd call it `nvtxTraceTaskGpuOwnership`, and also add that in the doc (that is going to show up in an NVTX range).

I'd make the val follow the naming pattern.

Reply:
> I'd make the val follow the naming pattern.

I can change the name. Can you elaborate more on the naming pattern?

Reply:
I just think we should say NVTX somewhere in the name. You could make it `spark.rapids.sql.nvtx.traceTaskGpuOwnership`, so we can start some sort of nvtx namespace.

Reply:
Changed the name as suggested.